pgwidget/pgwidget_core.py

```diff
@@ -839,6 +862,37 @@ def get_row_and_col_of_cell(self,cell):
         row_index=index%(self.rows+1)
         col_index=index//(self.rows+1)
         return(row_index,col_index)
+
+    def resize_table_by_rows_and_cols(self, rows, cols):
+        #will cut/add cols to the right and cut/add rows to the bottom
+
+
+        self.table_cells = []
+        self.cols = cols
+        self.rows = rows
+
+        self._initialize_cells()
+
+
+        self.table_size = [(self.cell_size[0] + self.margin) * self.cols + int(self.has_scrollbar) * 15, (self.cell_size[1] + self.margin) * (self.rows + 1)]
+
```

Comment:
15 = magical constant?

Reply:
Fix everywhere please

Reply:
The same initialization of self.table_size is used in the __init__. Should I refactor that everywhere and make the scrollbar width a variable?